cluster/src/test/scala/org/apache/pekko/cluster/protobuf/ClusterMessageSerializerSpec.scala

```diff
@@ -131,7 +131,7 @@ class ClusterMessageSerializerSpec extends PekkoSpec("pekko.actor.provider = clu
       checkSerialization(InternalClusterAction.Welcome(uniqueAddress, g2))
     }
 
-    // can be removed in 2.6.3 only checks deserialization with new not yet in effect manifests for 2.6.2
+    // can be removed in Akka 2.6.3 only checks deserialization with new not yet in effect manifests for Akka 2.6.2
```

Comment:
Should rephrase comment, i.e. remove in Pekko 1.2.0???

Reply:
My preference is to start by making the confusing version numbers explicit. There will be arguments about what Pekko versions will be the ones where we can remove code. There is an incomplete discussion on the mailing list. When we start making some decisions, we can go back into the code and replace the Akka versions with the agreed Pekko versions.